examples/titanic/titanic.py

```diff
@@ -74,7 +74,7 @@ def gen_family_size(self, data):
     @require('data_df')
     @will_generate('h5py', ['age', 'sibsp'])
     def gen_age_sibsp(self, data):
-        data_df = data['data_df'].copy()
+        data_df = data['data_df'].copy() # mutable
```

Comment:
why did you think people will know what you are thinking by just saying "mutable"?